src/main.tsx

```
@@ -2,15 +2,18 @@ import { StrictMode } from 'react';
 import { createRoot } from 'react-dom/client';
 import { QueryClientProvider } from '@tanstack/react-query';
 import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
+import { OverlayProvider } from 'overlay-kit';
 import { queryClient } from './shared/apis/queryClient.ts';
 import App from './App.tsx';
 import '@/shared/styles/global.css.ts';
 
 createRoot(document.getElementById('root')!).render(
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)